### PR TITLE
CI for private Releases

### LINKED ISSUE #765: Figure out CI for Private Releases without contract release
# Rationale

# Implementation

### PR DESCRIPTION
**Pull request by shyam-patel-kira**
fixes #765

### CODE REVIEW

File: export.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```
Comment (ashish10677) [inline export.sh:1]: Instead of adding a new file, can you modify generate-bindings.sh?

File: export.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  "StakedToken stakedToken.go"
+  "RAZOR RAZOR.go"
+  "AccessControl accessControl.go"
+  "ERC20 erc20.go"
+  "ERC165 erc165.go"
+)
+
+commitId=$1
+rm -rf contract-abi
+git clone https://github.com/razor-network/contracts.git
```
Comment (ashish10677) [inline export.sh:34]: This will clone the contracts in razor-go itself right?

**Review by ashish10677 — CHANGES_REQUESTED** [review]

File: export.sh (hunk 2 above)
Comment (shyam-patel-kira) [reply export.sh:34]: Yes it will clone contracts export abis for the given commitId and remove it

File: export.sh (hunk 1 above)
Comment (shyam-patel-kira) [reply export.sh:1]: okay, can modify it for public and private releases

**Pull request closed without merging**